core/src/oracle_config.rs

```diff
@@ -107,9 +107,9 @@ impl OracleConfig {
         )?;
         let refresh_box_wrapper_inputs = RefreshBoxWrapperInputs::build_with(
             bootstrap.refresh_contract_parameters.clone(),
-            token_ids.refresh_nft_token_id.clone(),
             token_ids.oracle_token_id.clone(),
-            token_ids.reward_token_id.clone(),
+            token_ids.pool_nft_token_id.clone(),
```

Comment:
Damn, I think I should have picked up on this in a prior review. As you say, it's why we need #47.

Reply:
Yes, #47 cannot come soon enough.